### PR TITLE
[EASY] add LST LRT price

### PR DESCRIPTION
**Pull request by GoodDataBadGuy**
# Thank you for contributing to Spellbook!

Thank you for taking the time to submit code in Spellbook. A few things to consider:

- If you are a first-time contributor, please sign the CLA by copy & pasting **_exactly_** what the bot mentions in PR comment
- Refer to [docs](#spellbook-contribution-docs) section below to answer questions
- Dune team will review submitted PRs as soon as possible

## Spellbook sub-projects

To prepare for Spellbook to scale in size, Dune team has started to pilot sub-projects:

- For a quick overview, see [the GH discussion on the topic](https://github.com/duneanalytics/spellbook/discussions/5238)
  - Please ask questions and continue the conversation there
- The first spell lineage to move: `tokens.erc20` and all upstream spells which help build it
  - **_Note_**: Please navigate to [tokens](/tokens) root directory to find tokens objects

## Spellbook contribution docs

The [docs](docs) directory has been implemented to answer as many questions as possible. Please take the time to reference each `.md` file within this directory to understand how to efficiently contribute & why the repo is designed as it is 🪄

Example questions to be answered:

- What does each property in the [model config block](/docs/models/model_config_block.md) mean?
- What is the [CI test](/docs/ci_test/ci_test_overview.md) attached to PRs and how can I best utilize it?
- What are the Spellbook [best practices](/docs/general/best_practices.md)?

Please navigate through the [docs](/docs) directory to find as much info as you can.

**Note**: happy to take PRs to improve the docs, let us know 🤝

### CODE REVIEW

File: models/prices/ethereum/prices_ethereum_tokens.sql [hunk 1]
```diff
@@ -1645,7 +1645,13 @@ FROM
     ('meth-meth', 'ethereum', 'mETH', 0xd5f7838f5c461feff7fe49ea5ebaf7728bb0adfa, 18),
     ('usde-ethena-usde', 'ethereum', 'USDe', 0x4c9EDD5852cd905f086C759E8383e09bff1E68B3, 18),
     ('ajna-ajna-protocol', 'ethereum', 'AJNA', 0x9a96ec9b57fb64fbc60b423d1f4da7691bd35079, 18),
-    ('ata-automata', 'ethereum', 'ATA', 0xA2120b9e674d3fC3875f415A7DF52e382F141225, 18)
+    ('ata-automata', 'ethereum', 'ATA', 0xA2120b9e674d3fC3875f415A7DF52e382F141225, 18),
+    ('oseth-stakewise-staked-eth','ethereum','osETH',0xf1c9acdc66974dfb6decb12aa385b9cd01190e38, 18),
+    ('lseth-liquid-staked-ethereum','ethereum','LsETH',0x8c1bed5b9a0928467c9b1341da1d7bd5e10b6549, 18),
```
Comment (GoodDataBadGuy) [inline models/prices/ethereum/prices_ethereum_tokens.sql:1650]: https://etherscan.io/address/0x8c1bed5b9a0928467c9b1341da1d7bd5e10b6549
https://coinpaprika.com/coin/lseth-liquid-staked-ethereum/

File: models/prices/ethereum/prices_ethereum_tokens.sql [hunk 2]
```diff
@@ -1645,7 +1645,13 @@ FROM
...
+    ('oeth-origin-ether','ethereum','OETH',0x856c4Efb76C1D1AE02e20CEB03A2A6a08b0b8dC3, 18),
```
Comment (GoodDataBadGuy) [inline models/prices/ethereum/prices_ethereum_tokens.sql:1651]: https://etherscan.io/address/0x856c4Efb76C1D1AE02e20CEB03A2A6a08b0b8dC3
https://coinpaprika.com/coin/oeth-origin-ether/

File: models/prices/ethereum/prices_ethereum_tokens.sql [hunk 3]
```diff
@@ -1645,7 +1645,13 @@ FROM
...
+    ('ethx-stader-ethx','ethereum','ETHx',0xA35b1B31Ce002FBF2058D22F30f95D405200A15b, 18),
```
Comment (GoodDataBadGuy) [inline models/prices/ethereum/prices_ethereum_tokens.sql:1652]: https://etherscan.io/address/0xA35b1B31Ce002FBF2058D22F30f95D405200A15b
https://coinpaprika.com/coin/ethx-stader-ethx/

File: models/prices/ethereum/prices_ethereum_tokens.sql [hunk 4]
```diff
@@ -1645,7 +1645,13 @@ FROM
...
+    ('rseth-rseth','ethereum','rsETH',0xa1290d69c65a6fe4df752f95823fae25cb99e5a7, 18),
+    ('pufeth-pufeth','ethereum','pufETH',0xD9A442856C234a39a81a089C06451EBAa4306a72, 18)
```
Comment (GoodDataBadGuy) [inline models/prices/ethereum/prices_ethereum_tokens.sql:1654]: https://etherscan.io/address/0xD9A442856C234a39a81a089C06451EBAa4306a72
https://coinpaprika.com/coin/pufeth-pufeth/

File: models/prices/ethereum/prices_ethereum_tokens.sql [hunk 5]
```diff
@@ -1645,7 +1645,13 @@ FROM
     ('meth-meth', 'ethereum', 'mETH', 0xd5f7838f5c461feff7fe49ea5ebaf7728bb0adfa, 18),
     ('usde-ethena-usde', 'ethereum', 'USDe', 0x4c9EDD5852cd905f086C759E8383e09bff1E68B3, 18),
     ('ajna-ajna-protocol', 'ethereum', 'AJNA', 0x9a96ec9b57fb64fbc60b423d1f4da7691bd35079, 18),
-    ('ata-automata', 'ethereum', 'ATA', 0xA2120b9e674d3fC3875f415A7DF52e382F141225, 18)
+    ('ata-automata', 'ethereum', 'ATA', 0xA2120b9e674d3fC3875f415A7DF52e382F141225, 18),
+    ('oseth-stakewise-staked-eth','ethereum','osETH',0xf1c9acdc66974dfb6decb12aa385b9cd01190e38, 18),
+    ('lseth-liquid-staked-ethereum','ethereum','LsETH',0x8c1bed5b9a0928467c9b1341da1d7bd5e10b6549, 18),
+    ('oeth-origin-ether','ethereum','OETH',0x856c4Efb76C1D1AE02e20CEB03A2A6a08b0b8dC3, 18),
+    ('ethx-stader-ethx','ethereum','ETHx',0xA35b1B31Ce002FBF2058D22F30f95D405200A15b, 18),
+    ('rseth-rseth','ethereum','rsETH',0xa1290d69c65a6fe4df752f95823fae25cb99e5a7, 18),
```
Comment (GoodDataBadGuy) [inline models/prices/ethereum/prices_ethereum_tokens.sql:1653]: https://etherscan.io/address/0xa1290d69c65a6fe4df752f95823fae25cb99e5a7
https://coinpaprika.com/coin/rseth-rseth/

**Review by jeff-dude — APPROVED** [review]
thank you!

**Pull request merged**